pycbc/io/live.py

```diff
@@ -310,6 +319,14 @@ def save(self, fname):
             logging.info('P_astro file saved as %s', self.pastro_file)
         return
```

Comment:
Ah, it seems you need to move this 'return' otherwise the code will just terminate the method here without reaching the embright saving block.